.gitignore

```diff
@@ -0,0 +1,3 @@
+**/node_modules/**
+.env
+package-lock.json
```

Comment:
Future cleanup note: If you're using `yarn`, this file should not be created. If you're using `npm`, this file should be checked in.

Reply:
I could use help figuring out the "infra" for this, let's talk offline